src/midiToNotes.js

```diff
@@ -88,9 +88,14 @@ const MidiToNotes = (function () {
       const bType = getEventType(b);
 
       // Tempo change events
-      if (aType === "meta" && a.metaType == 81) aPriority = 2;
-      if (bType === "meta" && b.metaType == 81) bPriority = 2;
-
+      if (aType === "meta" && a.metaType === 81) {
+        aPriority = 2;
+        if (a.deltaTime === 0) a.deltaTime = [b.deltaTime, b.deltaTime = a.deltaTime][0];
```

Comment:
Does this swap the deltaTimes? Why do we need to do this? Needs to be in a comment.

Reply:
Determined why these need to be swapped:

https://github.com/TC-Chart-Converter/TC-Chart-Converter.github.io/pull/11#issuecomment-2453118737